workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityGameObjectCommandHandlersGenerator.tt

```diff
@@ -97,6 +100,20 @@ namespace <#= qualifiedNamespace #>
                     return request.RequestId;
                 }
 
+                public long Send<#= commandDetails.CommandName #>Request(EntityId entityId, <#= commandDetails.FqnRequestType #> payload,
+                    Action<<#= receivedResponseType #>> callback, uint? timeoutMillis = null, bool allowShortCircuiting = false)
+                {
+                    if (!IsValid())
+                    {
+                        return -1;
```

Comment:
Quick refresher - why do return an invalid request id instead of throwing here?

Reply:
I'm not entirely sure. The effect is that it will send the command but not give you and ID which seems wrong. I don't think I changed the behaviour here though. So this can be fixed as part of the subscriptions work.

Reply:
Yeah this is the same behaviour - just not sure why we are doing C style errors